MetaMask/metamask-extension
# UX Multichain: Added avatar group component

## **Description**
This PR is to add a avatar group component required for the [avatars in account-list-item](https://github.com/MetaMask/MetaMask-planning/issues/1425)

## **Manual testing steps**

_1. Step1:_ Go to Storybook build
_2. Step2:_ Look for AvatarGroup Component
_3. Step3:_ Check if components are in a stacked manner
_4. Step4:_ Change the limit and check extra appear in a text like +2 format


## **Screenshots/Recordings**
![Screenshot 2023-10-12 at 4 29 52 PM](https://github.com/MetaMask/metamask-extension/assets/39872794/031c01ea-8665-48bd-991d-61ba40fc2b75)


## **Related issues**

_Fixes #21323 

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
